client/state/data-layer/wpcom/checklist/index.js

```diff
@@ -13,10 +13,39 @@ import { receiveSiteChecklist } from 'state/checklist/actions';
 
 import { registerHandlers } from 'state/data-layer/handler-registry';
 
-// The checklist API requests use the http_envelope query param, however on
-// desktop the envelope is not being unpacked for some reason. This conversion
-// ensures the payload has been unpacked.
-const fromApi = ( payload ) => get( payload, 'body', payload );
+// A list of known tasks for the calypso client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js
+const knownTaskIds = [
+	'site_created',
+	'domain_verified',
+	'email_verified',
+	'blogname_set',
+	'mobile_app_installed',
+	'site_launched',
+	'front_page_updated',
+	'site_menu_updated',
+];
```

Comment:
This feels like it should be a constant referenced somewhere

Reply:
You are right, I will DRY it up